### PR TITLE
Correct swerve configuration

### PR DESCRIPTION
**Pull request by rutmanz**

### CODE REVIEW

File: src/main/java/org/team1540/robot2024/util/swerve/SwerveFactory.java [hunk 1]
```diff
@@ -56,6 +55,27 @@ private SwerveModuleHW(int id, SwerveCorner corner, String canbus) {
...
+                    break;
+                case BACK_LEFT:
+                    invertTurn = true;
+                    invertDrive = false;
+                    break;
+                case BACK_RIGHT:
+                    invertTurn = false;
+                    invertDrive = true;
+                    break;
+            }
```
Comment (mimizh2418) [inline src/main/java/org/team1540/robot2024/util/swerve/SwerveFactory.java:77]: ew.

**Review by mimizh2418 — APPROVED** [review]
LGTM

**Comment by mimizh2418** [comment]
We should also look into updating the PIDF constants for the krakens

File: src/main/java/org/team1540/robot2024/util/swerve/SwerveFactory.java (hunk 1 above)
Comment (rutmanz) [reply src/main/java/org/team1540/robot2024/util/swerve/SwerveFactory.java:77]: would you prefer I use a hashmap

**Pull request merged**